tests/foreman/api/test_rhc.py

```diff
@@ -32,6 +32,8 @@ def fixture_enable_rhc_repos(target_sat):
 
 
 @pytest.mark.e2e
+@pytest.mark.pit_server
+@pytest.mark.pit_client
```

Comment:
As per my understanding,
In order to use `pit_server` & `pit_client` marker test should satiesfy some conditions
```
server tests will contains-  non destructive, not fips
client tests will contains- `--no-containers` marker, not fips
```
- If we look this test, it uses `destructive` marker, we can't use `pit_server` marker here.
- Talking about `pit_client` scenario, so test doesn't have anything related to rhel content host, good to remove client marker.

Reply:
Correct, I agree with Vijay.

Reply:
One small correction, `--no-containers` is a parameter that controls whether content hosts will be containers or VMs. If the test is already marked with `@pytest.mark.no_containers`, that is fine. In the PIT pipeline, we force all tests to run on VM by using `--no-containers` as a pytest argument.